containers/jenkins-master/config/jobs/os-gate-rc-stable/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.plugins.flow.BuildFlow plugin="build-flow-plugin@0.18">
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <com.sonyericsson.rebuild.RebuildSettings plugin="rebuild@1.25">
+      <autoRebuild>false</autoRebuild>
+      <rebuildDisabled>false</rebuildDisabled>
+    </com.sonyericsson.rebuild.RebuildSettings>
+  </properties>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders/>
+  <publishers/>
+  <buildWrappers/>
+  <icon/>
+  <dsl>parallel(
+	{ build("snappy-daily-rolling-openstack", channel: "candidate", type: "Promotion rc - stable") },
+    { build("snappy-rollbackstress-rolling-openstack", channel: "candidate", type: "Promotion rc - stable") }
+)
+// the upload job will probably be different for each gate
+build("github-snappy-daily-ppa")</dsl>
```

Comment:
I think this one should always be manually triggered. Even when we have the way to promote using snapcraft, somebody should trigger the job.